### PR TITLE
Create 01.1-Code_of_conduct.Rmd

### LINKED ISSUE #1: Add link to code of conduct
Prepare a PR adding subsection 1.x with the title "Code of Conduct" at the course page:
github.com/microbiome/course_2022_oulu

Which can be viewed at:
https://microbiome.github.io/course_2022_oulu/

Do as follows:
- Add the main text for CoC from here: https://bioconductor.github.io/bioc_coc_multilingual/
- Then add link to that page for more information (no need to copy contact info or other details)

I ask @shigdel to prepare the PR and @TuomasBorman can review.

### PR DESCRIPTION
**Pull request by shigdel**
added code of conduct file

### CODE REVIEW

File: 01.1-Code_of_conduct.Rmd [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Version 1.0.2 (July 27, 2021)
+
+The Bioconductor community values an open approach to science that promotes the
+
+    sharing of ideas, code, software and expertise
+    collaboration
+    diversity and inclusivity
+    a kind and welcoming environment
+    community contributions
+
```
Comment (TuomasBorman) [inline 01.1-Code_of_conduct.Rmd:15]: I'm not sure if this does bullets.

You could add 

   - line, I think it does it ('-')

File: 01.1-Code_of_conduct.Rmd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    Aggressive or browbeating behavior
+    Mocking or insulting another person’s intellect, work, perspective, or question/comment
+    Making reference to someone’s gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or other personal attributes in the context of a scientific discussion
+    Deliberately making someone feel unwelcome
+    Trolling behaviour (deliberately inflammatory or offensive messages)
+    Sustained off-topic posts
+
+Enforcement
+
```
Comment (TuomasBorman) [inline 01.1-Code_of_conduct.Rmd:47]: These could be bolded

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
You could add this all text to index.Rmd file. Check the book, index.Rmd contains 1 section

**Comment by TuomasBorman** [comment]
Thanks!

**Comment by antagomir** [comment]
Also @shigdel check locally if the rendering output looks OK. If unclear how to do, then more in Slack.

**Comment by TuomasBorman** [comment]
https://github.com/microbiome/course_2022_oulu#rendering-the-book

this shoudl do it

**Comment by shigdel** [comment]
> You could add this all text to index.Rmd file. Check the book, index.Rmd contains 1 section

i didnt see 1 section on index.Rmd

**Comment by antagomir** [comment]
It is in _bookdown.yml

**Comment by antagomir** [comment]
There is still a request. @shigdel can you take care of completing this one?

**Comment by antagomir** [comment]
Replaced by #5

**Pull request closed without merging**